so3/configs/refso3_ramfs_defconfig

```diff
@@ -88,35 +88,11 @@ CONFIG_MMU=y
 # SOO Mobile Entity - Smart Object Oriented subsystem
 #
 CONFIG_SOO_REFSO3=y
-# CONFIG_SOO_BLIND is not set
-# CONFIG_SOO_OUTDOOR is not set
-# CONFIG_SOO_AGENCY is not set
-# CONFIG_SOO_NET is not set
-# CONFIG_SOO_LEDCTRL is not set
-# CONFIG_SOO_CHAT is not set
-# CONFIG_SOO_WAGOLED is not set
-# CONFIG_SOO_IUOC is not set
-# CONFIG_SOO_SWITCH_KNX is not set
-# CONFIG_SOO_SWITCH_ENOCEAN is not set
-
 #
 # SOO Mobile Entity frontend drivers
 #
-CONFIG_VDUMMY_FRONTEND=y
+CONFIG_VDUMMY_FRONTEND=n
```

Comment:
Is there a difference between "=n" and "is not set" ?

Reply:
You're right, it should be "not set" which is automatically done by make. Here, I disabled manually, but this will change after a make refso3_ramfs_defconfig. So, I fix it with the right line.